src/libraries/System.Private.Xml.Linq/src/System/Xml/Linq/XComment.cs

```diff
@@ -74,7 +74,7 @@ public string Value
             }
             set
             {
-                if (value == null) throw new ArgumentNullException(nameof(value));
+                ArgumentNullException.ThrowIfNull(value);
                 bool notify = NotifyChanging(this, XObjectChangeEventArgs.Value);
                 this.value = value;
```

Comment:
(Dropping a marker.  I've made it this far, but have to take off for a bit, and will resume later)